ims/picasso/rest.py

```diff
@@ -110,3 +103,15 @@ def list_snapshots():
            [constants.IMAGE_NAME_PARAMETER])
 def remove_image():
     pass
+
+
+@rest_call("/create_disk", "PUT", constants.CREATE_DISK_COMMAND,
+           [constants.DISK_NAME_PARAMETER])
```

Comment:
Looks like we're missing a constants.DISK_NAME_PARAMETER in the create_disk API call.